config.h.cmake.in

```diff
@@ -262,6 +262,9 @@ are set when opening a binary file on Windows. */
 /* Define to 1 if you have the `fileno' function. */
 #cmakedefine HAVE_FILENO 1
 
+/* Define to 1 if you have the `H5Literate2' function. */
+#cmakedefine HAVE_H5LITERATE2
```

Comment:
Sorry, I should have remembered this...

Reply:
No worry it took me a little bit before I could square why cmake builds were working and autoconf wasn't. My first incorrect guess was compiler optimizations.